src/instructlab/utils.py

```diff
@@ -239,6 +244,58 @@ def git_clone_checkout(
     return repo
 
 
+def get_confluence_docs(
+    ctx,
+    source: Dict[str, Union[str, List[str]]],
+) -> List[str]:
+    """
+    Retrieve the content of Confluence documents
+
+    Args:
+        source (dict): Source info containing references
+
+    Returns:
+         List[str]: List of document contents.
+    """
+    cfg = ctx.obj.config.confluence
```

Comment:
You have declared `confluence` as an optional value. That means `cfg` can be `None`. You need to handle the case here to avoid `AttributeError`.